docs/configure_android_sdk.md

```diff
@@ -0,0 +1,179 @@
+ 
+   ```kotlin
+      // Adds an attribute to all future RUM events
+      GlobalRum.addAttribute(key, value)
+ 
+      // Removes an attribute to all future RUM events
+      GlobalRum.removeAttribute(key)
+   ```
+
+### Track widgets
+ 
+Widgets are not automatically tracked with the SDK. To send UI interactions from your widgets manually call Datadog API. [See example][7].
+
+
+## Configure Library Initialization
+ 
+The following methods in `DatadogConfig.Builder` can be used when creating the Datadog Configuration to initialize the library:
+ 
+| Method                           | Description                                                                                                                                                                                                                                                             |
+|----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `setServiceName(<SERVICE_NAME>)` | Set `<SERVICE_NAME>` as default value for the `service` [standard attribute][4] attached to all logs sent to Datadog (this can be overriden in each Logger).                                                                                                                                                           |
+| `setRumEnabled(true)`     | Set to `true` to enable sending RUM data to Datadog.                                                                                                                                                                                                                                  |
+| `trackInteractions(Array<ViewAttributesProvider>)` | Enables tracking User interactions (such as Tap, Scroll or Swipe). The parameter also allow you to add custom attributes to the RUM Action events based on the widget with which the user interacted. |
+| `useViewTrackingStrategy(strategy)` | Defines the strategy used to track Views. Depending on your application's architecture, you can choose one of several implementations of [`ViewTrackingStrategy`][3] or implement your own. |
+| `addPlugin(DatadogPlugin, Feature)`   | Adds a plugin implementation for a specific feature (CRASH, LOG, TRACE, RUM). The plugin will be registered once the feature is initialized and unregistered when the feature is stopped. |
+ 
+### Automatically Track Views
```

Comment:
I see that you have a link bellow for <Link to View Tracking Strategy>. That means that you will move this part there eventually ?